runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -119,15 +121,27 @@
 	private static final Pattern NUMBER_PATTERN = Pattern.compile("([0-9]+)");
 	private static final Pattern LEVEL_UP_PATTERN = Pattern.compile(".*Your ([a-zA-Z]+) (?:level is|are)? now (\\d+)\\.");
 	private static final Pattern BOSSKILL_MESSAGE_PATTERN = Pattern.compile("Your (.+) kill count is: <col=ff0000>(\\d+)</col>.");
+
+	private static final List<Pattern> PVP_KILL_MESSAGES = Arrays.asList(
+			Pattern.compile("With a crushing blow you finish (.+)\\."),
+			Pattern.compile("What was (.+) thinking challenging you\\.\\.\\."),
+			Pattern.compile("(.+) was no match for you\\."),
```

Comment:
You're missing "into tiny pieces and sat on them"

Reply:
Do you know the full string?

“You smashed [name] into tiny pieces and sat on them?”

Reply:
Just got a reply from Ash

https://twitter.com/JagexAsh/status/1056605406947143682

A lot more than I expected - not sure how to code for the gender one?